Description: Pass "model" when building .p.o files from assembler files.
 The upstream build system passed -DMODEL_$(MODEL) when building .o files but
 not when building .p.o files. When building for raspbian (with the correct
 v6 configuration) this resulted in arm.p.o coming out with v7 code.
Author: Peter Michael Green <plugwash@raspbian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- ocaml-4.01.0.orig/asmrun/Makefile
+++ ocaml-4.01.0/asmrun/Makefile
@@ -180,7 +180,7 @@ clean::
           exit 2; }
 
 .S.p.o:
-	$(ASPP) -DSYS_$(SYSTEM) $(ASPPPROFFLAGS) -o $*.p.o $*.S
+	$(ASPP) -DSYS_$(SYSTEM) -DMODEL_$(MODEL) $(ASPPPROFFLAGS) -o $*.p.o $*.S
 
 .c.d.o:
 	ln -s -f $*.c $*.d.c
